packages/hawtio/src/core/core.ts

```diff
@@ -3,6 +3,17 @@ import $ from 'jquery'
 import { eventService } from './event-service'
 import { log } from './globals'
 
+export interface HeaderItems {
+  // The components that should be populated as
+  // dropdown items on the header bar
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  components: React.ComponentType<any>[]
+
+  // Should components remain visible on header even when
+  // the plugin is not being displayed.
+  universal: boolean
+}
```

Comment:
We can instead define `HeaderItem` type as follows:
```
export type HeaderItem = React.ComponentType<any> | {
  component: React.ComponentType<any>
  universal: boolean
}
```
This way you can define a mix of local and universal header items within a plugin. The current fix only allows either all local or all universal.